src/main/java/StrEx10.java

```diff
@@ -0,0 +1,41 @@
+public class StrEx10 {
+    public String daoXau(String s) {
+        String s1 = "";
+        for (int i = 0; i < s.length(); i++) {
+            s1 += s.charAt(s.length() - 1 - i);
+        }
+        return s1;
+    }
+
+    public String sum(String s1, String s2) {
+        StringBuffer s = new StringBuffer();
```

Comment:
```suggestion
        StringBuffer buffer = new StringBuffer();
```

Đưa khai báo xuống dưới